cpp/tests/groupby/lists_tests.cu

```diff
@@ -194,5 +194,19 @@ TYPED_TEST(groupby_lists_test, lists_with_nulls)
   test_sum_agg(keys, values, expected_keys, expected_values);
 }
 
+TYPED_TEST(groupby_lists_test, lists_with_null_elements)
+{
+  auto keys   = lcw<TypeParam>{{{{1, 2, 3}, {}, {4, 5}, {}, {6, 0}},
+                              {{1, 2, 3}, {}, {4, 5}, {}, {6, 0}},
+                              {{1, 2, 3}, {}, {4, 5}, {}, {6, 0}},
+                              {{1, 2, 3}, {}, {4, 5}, {}, {6, 0}}},
+                             nulls_at({2, 3})};
```

Comment:
Correct me if I'm wrong but this still looks like a list column with entire row null rather than only certain elements being null.

Reply:
An example print of keys column:
```
 List<List<int32_t>>:
 Length : 4
 Offsets : 0, 5, 10, 10, 10
 Null count: 2
 0011
    List<int32_t>:
    Length : 10
    Offsets : 0, 3, 3, 5, 5, 7, 10, 10, 12, 12, 14
       1, 2, 3, 4, 5, 6, 0, 1, 2, 3, 4, 5, 6, 0
```

Reply:
Updated.